lib/Statistics/GATracking.php

```diff
@@ -68,4 +66,13 @@ private function addParams($extracted_links, $ga_campaign, $internal_host = null
     }
     return $processed_links;
   }
+
+  private function splitLink($link) {
+    $parts = explode('#', $link);
+    $hash = implode('#', array_slice($parts, 1));
```

Comment:
Btw. there is also `explode('#', $link, 2)` which will split a string to max. 2 parts by the first occurrence of `#`. I guess no need to change it now but it's a handy parameter to know about.

Reply:
Cool, I didn't know about it.